### ISSUE TITLE
Split our own Plutip tests up / improve test speed

### ISSUE THREAD

**Issue by ngua** [Plutonomicon/cardano-transaction-lib#1011]
CI is taking a long time in part because we are running over 40 tests with Plutip sequentially. We can split these up so they build in parallel

**Comment by klntsky** [comment]
There will be port conflicts if we don't specify multiple configs.

**Comment by klntsky** [comment]
Also in case we run the tests in parallel, logging will become too complicated. I think we can remove some of the more simple tests

**Comment by jy14898** [comment]
I think we can improve test speed by keeping all the processes alive between tests in a group (via mote's bracket) and sending stop/start requests to the plutip cluster to create the necessary wallets per test.

I'm revisiting my plutip process management PR to also fix #627, so I can integrate the changes there.

**Comment by klntsky** [comment]
@jy14898 We can group tests that do not rely on specific utxo distributions (most of them I think). Good idea!
Also note the new [spawnAndCollectOutput](https://github.com/Plutonomicon/cardano-transaction-lib/blob/7f9c1d88a0d704eefa7a1fd1be4ff7c3a407d4e2/src/Internal/Test/E2E/Runner.purs#L621) function in [my recent PR ](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1075/files#) - I think we should use it to avoid duplicate work, because what it does matches is exactly what is needed here.
